modes/zh/pipeline.md

```
@@ -0,0 +1,64 @@
+
+**特殊情况处理：**
+- **LinkedIn**：可能会触发强登录校验拦截。若多次失败，标记为 `[!]` 并引导候选人直接粘贴 JD 文本。
+- **PDF 文件**：如果 URL 链接直接指向一份 PDF 文档，使用 Read 工具读取二进制文档。
+- **本地文件前缀 `local:`**：如果是本地路径，直接读取对应文件。例如：`local:jds/linkedin-pm-ai.md` 指向读取本地的 `jds/linkedin-pm-ai.md` 文件。
+
+---
+
+## 数据一致性校验
+
+在启动处理之前，先执行以下脚本检查简历与配置的同步状态：
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)